content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -271,61 +315,73 @@ When you checkout the metadata on your newly created NFT, you should now see a
 The last thing you need to do is verify the NFT. This effectively just flips the
 `verified` field above to true, but it's incredibly important. This is what lets
 consuming programs and apps know that your NFT is in fact part of the
-collection. You can do this using the `verifyCollection` function:
+collection. You can do this using the `verifyCollectionV1` function:
```

Comment:
```suggestion
collection - because the Collection's owner has signed a transaction making the NFT a member of that collection. You can do this using the `verifyCollectionV1` function:
```